migrating_from_ocp_3_to_4/premigration-checklists-3-4.adoc

```diff
@@ -84,10 +87,20 @@ NFS does not require a defined storage class.
 If an application uses an internal image in the `openshift` namespace that is not supported by {product-title} {product-version}, you can manually update the xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-updating-deprecated-internal-images_troubleshooting-3-4[{product-title} 3 image stream tag] with `podman`.
 * [ ] The target cluster and the replication repository have sufficient storage space.
 * [ ] The xref:../authentication/understanding-identity-provider.adoc#supported-identity-providers[identity provider] is working.
+* [ ] DNS records for your application exist on the target cluster.
 * [ ] Set the value of the `annotation.openshift.io/host.generated` parameter to `true` for each {product-title} route to update its host name for the target cluster. Otherwise, the migrated routes retain the source cluster host name.
+* [ ] Certificates that your application uses exist on the target cluster.
+* [ ] You have configured appropriate firewall rules on the target cluster.
+* [ ] You have correctly configured load balancing on the target cluster.
+* [ ] If a namespace already exists on the target cluster that has the same name as a namespace being migrated from the source, you have made sure that there are no objects in the target namespace that are of the same type and have the same name as objects being migrated.
```

Comment:
Nice, this is long but it makes sense to me and I have no idea where to start if you wanted it shorter :slightly_smiling_face: LGTM

Reply:
Suggestion: 

If you migrate objects to an existing namespace on the target cluster, the target namespace must contain no objects of the same name or type as the objects being migrated.